spec/request/messages/create_spec.rb

```diff
@@ -34,5 +34,17 @@
       subject
       expect(NotifyRequestJob).to have_been_enqueued
     end
+
+    context 'when unread messages on the other user conversation' do
+      it 'returns correct amount of unread messages on conversation' do
+        get api_v1_conversations_path, headers: another_auth_user_headers, as: :json
+        expect(parsed_data['conversations'].first['unread_messages'])
+          .to eq(0)
+        subject
+        get api_v1_conversations_path, headers: another_auth_user_headers, as: :json
+        expect(parsed_data['conversations'].first['unread_messages'])
+          .to eq(1)
+      end
```

Comment:
It's not a good idea to call the endpoint twice in the same it. If you need to set a state before a call, use a context. Let me know if it's not clear.